tests/conftest.py

```diff
@@ -243,21 +244,25 @@ def general_stream() -> Dict[str, Any]:
         "audible_notifications": False,
         "description": "General Stream",
         "rendered_description": "General Stream",
-        "is_old_stream": True,
```

Comment:
I could not find any documentation in the api docs or CZO discussion for this field. What was the field's original purpose, if you know? I can then add it's reason for removal in the commit text with more context.

Reply:
It's in the API changelog (https://zulip.com/api/changelog), and I found it on czo to be zulip/zulip#15409.